frontend/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -60,6 +49,20 @@ const applySortIndicator = (_columns, field, dir) =>
   }));
 
 export class NoticeOfWorkTable extends Component {
+  transformRowData = (applications) =>
```

Comment:
Is there a reason this is moved inside the class?

Reply:
It uses `this.props`, and since it's already a class component, I just decided to move the function onto the class instead of passing the params around.